### PR TITLE
Chapter 79 implementation

### PR DESCRIPTION
**Pull request by febincp6**

### CODE REVIEW

**Review by faisalmohd83 — CHANGES_REQUESTED** [review]
@febincp6 found the CDATA is missing one square bracket in the end `</p>]]>`. Please refer to previous chapters.

**Comment by faisalmohd83** [comment]
@hafizkvpg @sajadhaja Can we add GitHub actions to check the HTML/XML lint?

**Comment by sajadhaja** [comment]
Lint issues can be check using ES Lint plugin which is very popular.We can configure eslint rules and using npm manager we can execute the lint command to verify those configured rules.
Reference:-
ES Lint plugin: https://www.npmjs.com/package/eslint-plugin-html

Answer to your question, I seen some plugin available in github market place to run such ESLint check. Finally those rules should be configured in our code base and using npm command they are executing those rules.
Reference for Github ES Lint Check:-
- https://github.com/marketplace/actions/eslint-checker
- https://github.com/ninosaurus/eslint-check

File: podcast.rss [hunk 1]
```diff
@@ -6968,7 +6968,7 @@
     <item>
       <title>592  സൂറഃ നാസിയാത്ത് ആമുഖം_a</title>
```
Comment (faisalmohd83) [inline podcast.rss]: Can we follow the same format in item title as we did for previous chapters?

` <title>590.സൂറഃ നാസിയാത്ത് പാരായണം Part 1</title>`
` <title>592  സൂറഃ നാസിയാത്ത് ആമുഖം_a</title>`

File: podcast.rss [hunk 2]
```diff
@@ -7012,7 +7012,7 @@
     <item>
       <title>596  സൂറഃ നാസിയാത്ത്  ആയത്ത്  6-9 part 1_a</title>
```
Comment (faisalmohd83) [inline podcast.rss]: Let's maintain consistency between words. For e.g., below title had more than single space between words.

`<title>597   സൂറഃ നാസിയാത്ത്   ആയത്ത് 5-9 Part 2_a</title>`

File: podcast.rss [hunk 3]
```diff
@@ -7144,7 +7144,7 @@
     <item>
       <title>608  സൂറഃ നാസിയാത്ത്  ആയത്ത് 20.21&22 Part 1_a</title>
```
Comment (faisalmohd83) [inline podcast.rss]: The `&` sign to be replaced with `&amp;` in titles, please check other occurrences too.

File: podcast.rss [hunk 4]
```diff
@@ -7243,7 +7243,7 @@
     <item>
       <title>617  സൂറഃ നാസിയാത്ത്  ആയത്ത് 30&31_a</title>
```
Comment (faisalmohd83) [inline podcast.rss]: Please remove **_a**, found in many other titles too

File: podcast.rss [hunk 5]
```diff
@@ -7199,7 +7199,7 @@
     <item>
       <title>613  സൂറഃ നാസിയാത്ത്  ആയത്ത് 25&26 Part 1_a</title>
```
Comment (faisalmohd83) [inline podcast.rss]: Please remove **_a**, found in many other titles too

File: podcast.rss [hunk 6]
```diff
@@ -7320,7 +7320,7 @@
     <item>
       <title>624  സൂറഃ നാസിയാത്ത്  ആയത്ത് 45_a</title>
       <description>
-        <![CDATA[<p>Velicham Quran Dars series</p><p>Qur'an class #624💠Surah An-Naziat 💠</p><p>*ആയത്ത്&zwnj; - 45* إِنَّمَآ أَنتَ مُنذِرُ مَن يَخْشَىٰهَا നിശ്ചയം നീ അതിനെ ഭയപ്പെടുന്നവർക്ക്&zwnj; താക്കീതു നൽകുന്നവൻ മാത്രമാണ്. 🔸Admin Team Velicham Onlive🔸</p>]>
+        <![CDATA[<p>Velicham Quran Dars series</p><p>Qur'an class #624💠Surah An-Naziat 💠</p><p>*ആയത്ത്&zwnj; - 45* إِنَّمَآ أَنتَ مُنذِرُ مَن يَخْشَىٰهَا നിശ്ചയം നീ അതിനെ ഭയപ്പെടുന്നവർക്ക്&zwnj; താക്കീതു നൽകുന്നവൻ മാത്രമാണ്. 🔸Admin Team Velicham Onlive🔸</p>]]>
```
Comment (faisalmohd83) [inline podcast.rss:7323]: An issue in the text to HTML conversion. Let's use https://wordhtml.com/ to convert the texts to Html with compress option. The compress option is avail in the second tab(HTML) of the https://wordhtml.com/ site.

The text pushed:
`<![CDATA[<p>Velicham Quran Dars series</p><p>Qur'an class #624💠Surah An-Naziat 💠</p><p>*ആയത്ത്&zwnj; - 45* إِنَّمَآ أَنتَ مُنذِرُ مَن يَخْشَىٰهَا നിശ്ചയം നീ അതിനെ ഭയപ്പെടുന്നവർക്ക്&zwnj; താക്കീതു നൽകുന്നവൻ മാത്രമാണ്. 🔸Admin Team Velicham Onlive🔸</p>]]>`

--
A sample converted for reference with compress option.
`<p>Velicham Qur&rsquo;an Dars series</p><p>Qur'an class #624</p><p>💠Surah An-Naziat 💠<br />ആയത്ത്&zwnj; - 45</p><p>إِنَّمَآ أَنتَ مُنذِرُ مَن يَخْشَىٰهَا <br /> നിശ്ചയം നീ അതിനെ ഭയപ്പെടുന്നവർക്ക്&zwnj; താക്കീതു നൽകുന്നവൻ മാത്രമാണ്.</p><p>🔸Admin Team Velicham Onlive🔸</p>`

**Review by faisalmohd83 — CHANGES_REQUESTED** [review]
@febincp6 please address the new comments

File: podcast.rss (hunk 6 above)
Comment (faisalmohd83) [reply podcast.rss:7323]: @febincp6 Can you look at this comment? It might be worth looking at each verse for the same issue.

**Review by faisalmohd83 — APPROVED** [review]

**Comment by faisalmohd83** [comment]
Closes #87

**Pull request merged**